_episodes/07-github.md

```diff
@@ -144,6 +144,7 @@ Branch master set up to track remote branch master from origin.
 > $ git config --global --unset https.proxy
 > ~~~
 > {: .bash}
+> Please remember that for using SSH as connection, proxy setup is not needed.
```

Comment:
Something as above. I think mentioning details of SSH won't be useful and may be confussing for most of our learners. We have a few links to set up properly SSH in this episode and I think this is enough.